### PR TITLE
Add card check

### LINKED ISSUE #294: PROJECT- CREDIT CARD CHECKER using C++
## 🚀 Feature

The project is used to check the authenticity of the credit card. This program follows **Luhn’s Algorithm** (which the most card follows). In this stage the program works on fewer number of cards...hope for collaborative work to expand it.

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

## Motivation

Made this program while learning how to implement basic algorithm. 

## Pitch

- Easy to use level-1 program for Desktop Application/C++.

- It's a console-based application that runs in a console window same as a C++ program.

- It doesn’t have any graphical user interface. Console Applications will have a character-based interface.

- So, anyone with just having a compiler to compile a C/C++ program can use this code.

### PR DESCRIPTION
**Pull request by Satyam26-09**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

YES

### Description

The project is used to check the authenticity of the credit card. This program follows Luhn’s Algorithm (which the most card follows). In this stage the program works on fewer number of cards...hope for collaborative work to expand it.

Fixes #294 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue: #294

PR made under GSSoC'22 contributions

### CODE REVIEW

**Review by photon149 — COMMENTED** [review]
@Satyam26-09 Please Add Output SS in your README file as well.

**Comment by Satyam26-09** [comment]
@photon149 Done....

**Review by photon149 — APPROVED** [review]
Looks good to me.
@prakhar3062 please have a look.

**Review by prakhar3062 — APPROVED** [review]
Everything works fine

File: Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        }
+        else
+        cout<<"INVALID";
+    }
+    else
+        cout<<"INVALID";
+        return 0;
+}
+
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp:60]: Delete this space !

File: Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    int i=0;
+    do
+    {
+        l=l/10;
+        i++;
+    } while (l!=0);
+        return i;
+}
+
+
```
Comment (Kushal997-das) [inline Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp:90]: Delete this space !

File: Desktop Application/Basic/C++/Credit Card Checker/Documents/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+This project is used to check the authenticity of the credit card. This program follows Luhn’s Algorithm (which the most card follows). In this stage the program works on fewer number of cards which are:- VISA , MasterCard & American Express. One who knows more algorithms can add to it easily.
```
Comment (Kushal997-das) [inline Desktop Application/Basic/C++/Credit Card Checker/Documents/README.md:1]: Change this file location .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@Satyam26-09 Your `readme.md` file should under[this](https://github.com/Satyam26-09/Project-Guidance/tree/Add_CardCheck/Desktop%20Application/Basic/C%2B%2B/Credit%20Card%20Checker) Instead [this](https://github.com/Satyam26-09/Project-Guidance/blob/Add_CardCheck/Desktop%20Application/Basic/C%2B%2B/Credit%20Card%20Checker/Documents/README.md) .

File: Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp (hunk 2 above)
Comment (Satyam26-09) [reply Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp]: done

File: Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp (hunk 1 above)
Comment (Satyam26-09) [reply Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp]: done..

File: Desktop Application/Basic/C++/Credit Card Checker/Documents/README.md (hunk 3 above)
Comment (Satyam26-09) [reply Desktop Application/Basic/C++/Credit Card Checker/Documents/README.md]: done..

Comment (Satyam26-09) [reply Desktop Application/Basic/C++/Credit Card Checker/Documents/README.md]: @Kushal997-das I have changed the file location simply by deleting and creating new file....is there any other way to do so...kindly share

File: Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp (hunk 1 above)
Comment (Satyam26-09) [reply Desktop Application/Basic/C++/Credit Card Checker/ccc.cpp]: @Kushal997-das I have used github to made changes in ccc.cpp as I was facing problem to do so with git bash...
Screenshot showing this problem..
kindly help...

**Review by Kushal997-das — APPROVED** [review]
@Satyam26-09 :+1:

**Pull request merged**